dotSyntax/src/main/scala/monocle/syntax/AppliedLens.scala

```diff
@@ -41,6 +41,9 @@ trait AppliedLens[A, B] extends AppliedOptional[A, B]{
   override def sixth(implicit ev: Field6[B]): AppliedLens[A, ev.B] =
     compose(ev.sixth)
 
+  override def reverse(implicit ev: Reverse[B]): AppliedLens[A, ev.B] =
```

Comment:
I think you also need to override it for `AppliedPrism`

Reply:
If I override it on `AppliedPrism` as
`  override def reverse(implicit ev: Reverse[B]): AppliedPrism[A, ev.B] =`
I start getting issues in `AppliedIso` as its parents `AppliedPrism` and `AppliedLens` both override `reverse` with different signatures. 
Any ideas?

Reply:
got it, it is because of `reverse` in `AppliedIso` returns an `AppliedLens` while it should be an `AppliedIso`.

Reply:
Thanks, I fixed that in the latest PR